extensions/timeseries/src/main/resources/ai/djl/timeseries/dataset/m5forecast_parser.json

```diff
@@ -0,0 +1,3100 @@
+{
```

Comment:
Maybe we should keep this in S3/metadata instead

Reply:
I think it's better keep it with the code:
1. This is tightly coupled with the code.
2. The s3 location become confusing, it doesn't make sense to keep it in unittest dataset
3. Users has to copy this file if they want to download dataset from Kaggle